.github/workflows/perl-tests.yml

```diff
@@ -30,13 +30,24 @@ jobs:
         run: |
           apt update
           apt install -y rsync default-mysql-server
+      - name: Get Perl Version
+        id: get-perl-version
+        run: |
+          echo version="$(perl -le 'print $]')" >> $GITHUB_OUTPUT
+        shell: bash
+      - name: Cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            /usr/local/lib/perl5/site_perl
+          key: ${{ runner.os }}-${{ steps.get-perl-version.outputs.version }}-${{ hashFiles('Makefile.PL') }}
       - name: Install prereqs (cpan)
         # This could probably be made more efficient by looking at what it's
         # installing via cpanm that could, instead, be installed from apt.  I
         # may do that later, but for now, it's fine! -- rjbs, 2023-01-07
         run: cpanm --notest --installdeps .
       - name: Run the tests
-        if: !(contains(github.event.pull_request.labels.*.name, 'want-coverage'))
+        if: "!(contains(github.event.pull_request.labels.*.name, 'want-coverage'))"
```

Comment:
does this belong in its own commit?  I think yes: not a showstopper but also easy to change

Reply:
Good catch.  It actually belongs in the first commit, not this one.

Reply:
(i.e. it was a yaml format error without the quotes, and apparently I didn't add it to the right one the first time.  It is now part of the first commit that adds coverage.)